### PR TITLE
Small tweaks

### PR DESCRIPTION
**Pull request by scottcwilson**

### CODE REVIEW

File: content/user/admin_pages/localization/currencies.md [hunk 1]
```diff
@@ -220,7 +220,7 @@ Rates that are correctly converted will be listed in green at the top of the pag
 Once you have verified that clicking the Update Currencies button operates correctly and without errors, you can optionally create a "cron job" in your hosting company's control panel to run the update automatically on a scheduled basis. The command to give it is as follows. Work with your hosting company's tech support team if you need help with determining the correct path and the correct php binary to call.
 
 ```code
-php /full/server/path/to/your_admin/currency_cron.php
+php /full/server/path/to/admin/currency_cron.php
```
Comment (drbyte) [inline content/user/admin_pages/localization/currencies.md:223]: Curious: why the change?

Comment (scottcwilson) [reply content/user/admin_pages/localization/currencies.md:223]: Trying to keep all "your_" as "your".  In this case, it doesn't read as well. 

See https://github.com/zencart/documentation/blob/master/CONTRIBUTING.md (search down for "your_")
